src/main/java/com/revature/controllers/SurveyController.java

```diff
@@ -29,6 +29,16 @@
 	public List<Survey> findAll() {
 		return surveyService.findAllOrderByDateCreatedDesc();
 	}
+	
+	@GetMapping("template")
+	public List<Survey> findAllTemplate() {
+		return surveyService.findAllTemplateOrderByDateCreatedDesc();
+	}
+	
+	@GetMapping("published")
```

Comment:
should be done similar to the template endpoint

Reply:
removed published endpoint